### ISSUE TITLE
Builder from RFTools causing issue with time

### ISSUE THREAD

**Issue by IyaTaisho** [MJRLegends/Space-Astronomy-2-Feedback-#256]
### ---Issue Report---
**(PLEASE DONT DELETE THIS TEMPLATE, YOUR ISSUE WILL BE IGNORED AND CLOSED IF YOU DO.)**

**Have you checked Open/Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING):** Yes

### Description of Issue
On 5/2, I got a chance to play in my world but noticed immediately some weirdness going on.

The last time I had played, I had been working on setting up my Attunement Altar for Astral Sorcery.  Prior to working on that, I had turned on my RFTools Builder in the Mining Dimension due that I was in need of Dimensional Shards.

I picked back up with my game spawning me on my Attunement Altar where I had quit.  I immediately noticed that I was having some kind of weird issues.  I tried placing down the Spectral Relays but the particle effects kept vanishing.  It was right at dark (7:30 - 8:00 PM), so it made sense that particle effects would be visible within this altar, but they kept vanishing.

Also,  I would break a Spectral Relay to pick it back up, but it took a few moments for it to go into my inventory (I have a magnet on so it should have immediately sucked into it).

Aside from that, I noticed the sun was forever hanging in the sky.  I glanced at the time and saw it doing a weird loop back.  It would start around 7:30, get up to about 7:45 or so, then jump back to 7:30.  It constantly kept doing it.

I couldn't figure out the issue and finally decided to fall back to a previous backup.  Everything seemed fine and I had no issues (other than my Flux Plug/Point issue).

I began retracing what I had previously done after that backup, which was go to the Mining Dimension and flip on my miner (RFTools Builder).  I instantly experienced the same issue as I described above (time began it's weird looping, I was breaking blocks and they didn't instantly break and when they did break, they didn't fly to me immediately, etc.).

I shut off my Builder and suddenly things went back to normal.  I wanted to narrow down the issue to whether it was the Mining Dimension or not, so I went home and then hopped into the Deep Dark where I have another RFTools Builder.  I flipped it on and again I had the same issues.

Prior to this, I could run either Builder and had no issue.  This just popped in this week.

### Steps to Reproduce Issue
1. Set up an RFTools Builder as a quarry in Deep Dark or Mining Dimension.
2. The set up I have is simple:
- RFTools Builder
- Advanced Powercell (100% infused)
- Lever (Builder is running off of Redstone signal)
- Ender Chest (white/white/white)
- Uncertain of dimensions of mining.  I'm thinking it was a large square to Bedrock with square being 110x110.  Something to that effect I think.  I'll double check.
3. Flip on Builder.
4. For me, the time looping starts up instantly.  Framerate remains the alright (bounces up and down like normal), movement is fine, just issues breaking blocks.

**Crash log/Log (if applicable):** [PASTEBIN. Do NOT copy and paste the log into the issue post itself.]
> *How do i locate a crash log on Minecraft on Curse/Twitch? see* https://gyazo.com/3c616a7dfcbe69033ea210805b093f82

**Screenshots (if applicable):** 

**Version of Mod Pack using (PLEASE DONT USE 'Latest' OR 'Newest'):** 1.4.9, 1.4.8 (rolled back to 1.4.7 originally in backup but upgraded back to 1.4.9 prior to playing).

**Where did this bug happen:** Single Player

### Additional Information
[Any other information that may be able to help me with the problem]
I'm going to test this in 1.4.7.  I don't think it happens there but not sure.  RFTools hasn't been upgraded, so what the issue could be is the million dollar question.

**UPDATE:**  I tested my world in 1.4.7 and do not get the issues I'm experiencing in 1.4.8 and 1.4.9.

**Comment by IyaTaisho** [comment]
I just tested my game on 1.5.0 and the same bug occurs.  Not sure what is up with the funky time issue.  The game seems to chug a little bit while the Builder is still running but back to normal when I shut it off.

If there are any details you would like from me, please let me know.  Thanks!

**Comment by MJRLegends** [comment]
Ok sorry for the delay for the this issue i need a profiler generated file on your world when its causing block breaking issues. If you still get this issue and post a profiler file i will reopen this and take a look for you.

Here are the details of the profiler
```
http://files.player.to/sampler-1.69.3.jar

Running Sampler 
Then you can /sampler start and /sampler stop while it's lagging.

Then a simple /sampler export [name of exported nps file here] will write out the .nps file that can be read by the developers in VisualVM.

Important: where to run Sampler? 
Is it a lag issue on the server, like tick lag? Run Sampler on the server, NOT on your client!

Is it a lag issue on the client, like FPS lag? Run Sampler on the client, NOT on your server!
```

**Comment by RundownRhino** [comment]
Update on this issue: I've had this problem(200ms/t+ TPS lag when using the quarry) and managed to discover the cause.
After profiling using Sampler, I saw this in the report:
![unknown](https://user-images.githubusercontent.com/52856631/67164221-cbad3380-f380-11e9-960e-23eaf7afe7bc.png)
It seems that the problem is causing by OpenSecurity making block breaking checks very costly. I solved this problem by removing OpenSecurity - not sure if there's another solution.
